src/test/resources/org/kohsuke/github/GHRepositoryTest/wiremock/addCollaborators/__files/orgs_github-api-test-org-a3d2b552-58b8-4028-b731-d00420496ca8.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
@martinvanzijl  It looks like you modified the test resources for this, whether you meant to or not.  This would be where the error is coming from.

Reply:
@bitwiseman Sorry, my mistake. Should be fixed now.